src/main/webapp/configure.js

```diff
@@ -259,8 +259,19 @@ async function loadMobileInfo(a, b, o, err) {
 function hideAndMoveAdvancedBody(_id) {
     const tBody = document.querySelector("#" + _id).parentNode; // initial advanced block content
     const initialAdvancedBlock = tBody.previousSibling; // advanced link button block and here was hidden the initial advanced block content
+    const advancedLink = initialAdvancedBlock.querySelector(".advancedLink");
     initialAdvancedBlock.querySelector(".advancedBody").appendChild(tBody); // moves the initial advanced block content back to the hidden block
```

Comment:
Please define a constant to be reused later, for example:
const advancedBody = advancedBlock.querySelector(".advancedBody");